promptsource/templates/clue/afqmc/templates.yaml

```diff
@@ -0,0 +1,69 @@
+dataset: clue
+subset: afqmc
+templates:
+  997437fd-6888-482d-95e9-ffd867b497ee: !Template
+    answer_choices: no ||| yes
+    id: 997437fd-6888-482d-95e9-ffd867b497ee
+    jinja: 'Do "{{ sentence1 }}" and "{{ sentence2 }}" express the same thing?
+
+      |||
+
+      {{ answer_choices[label] }}'
```

Comment:
Stupid question: How does this generate samples exactly? in particular with `\n` and `whitespaces` in the beginning and the end? Does it get trimmed all the time?

Reply:
Yeah the \n and whitespaces before & after `|||` get trimmed away